proxy/src/config.rs

```diff
@@ -52,6 +52,7 @@ pub struct TlsConfig {
 }
 
 pub struct HttpConfig {
+    pub accept_ws: bool,
```

Comment:
nit: I would call it `accept_websocket` since it's public config field. Up to you.